.github/workflows/pr-tests.yml

```diff
@@ -29,14 +29,18 @@ jobs:
         path: |
           ~/pilcom/node_modules
         key: ${{ runner.os }}-pilcom-node-modules
-    - name: Install Rust toolchain 1.72
-      run: rustup toolchain install 1.72-x86_64-unknown-linux-gnu
+    - name: Install Rust toolchain 1.72 (with clippy and rustfmt)
+      run: rustup toolchain install 1.72-x86_64-unknown-linux-gnu && rustup component add clippy --toolchain 1.72-x86_64-unknown-linux-gnu && rustup component add rustfmt --toolchain 1.72-x86_64-unknown-linux-gnu
     - name: Install nightly
       run: rustup toolchain install nightly-2023-01-03-x86_64-unknown-linux-gnu
     - name: Install riscv target
       run: rustup target add riscv32imac-unknown-none-elf --toolchain nightly-2023-01-03-x86_64-unknown-linux-gnu
     - name: Install stdlib
       run: rustup component add rust-src --toolchain nightly-2023-01-03-x86_64-unknown-linux-gnu
+    - name: Lint
+      run: cargo clippy --all --all-features -- -D warnings
```

Comment:
Is it more performant to group the calls with `--all-features` and `--no-default-features`, so that it does not have to go back-and-forth?